### PR TITLE
order flows alphabetically to facilitate selection and include creati…

### PR DESCRIPTION
**Pull request by norkans7**
…on date in the choice display

### CODE REVIEW

File: ureport/polls/tests.py [hunk 1]
```diff
@@ -468,7 +468,7 @@ def test_create_poll(self):
         with patch('dash.orgs.models.Org.get_flows') as mock:
```
Comment (rowanseymour) [inline ureport/polls/tests.py]: Am confused by this. Where is `get_flows` defined? I don't see it in https://github.com/rapidpro/dash/blob/master/dash/orgs/models.py

Comment (norkans7) [inline ureport/polls/tests.py]: those methods are really specific to ureport so there are not is dash but https://github.com/rapidpro/ureport/blob/order_flows/ureport/utils/__init__.py#L331

and they lookup the data from the cache which is updated by a task by calling 
https://github.com/rapidpro/ureport/blob/order_flows/ureport/utils/__init__.py#L292

File: ureport/utils/__init__.py [hunk 2]
```diff
@@ -306,6 +306,7 @@ def fetch_flows(org):
             if flow.rulesets:
                 flow_json = dict()
                 flow_json['uuid'] = flow.uuid
+                flow_json['created_on'] = flow.created_on.strftime('%Y-%m-%d')
```
Comment (rowanseymour) [inline ureport/utils/__init__.py]: If you think you might ever need the full `created_on` you could store it in the cache as full iso string and reduce it to just the date when rendering the PollForm

**Comment by rowanseymour** [comment]
Looks fine

**Pull request merged**